src/colmap/exe/colmap.cc

```diff
@@ -128,6 +128,8 @@ int main(int argc, char** argv) {
   commands.emplace_back("patch_match_stereo", &colmap::RunPatchMatchStereo);
   commands.emplace_back("point_filtering", &colmap::RunPointFiltering);
   commands.emplace_back("point_triangulator", &colmap::RunPointTriangulator);
+  commands.emplace_back("position_prior_mapper",
```

Comment:
I suggest calling this `pose_prior_mapper`, so we can generalize it to also consider rotations later without renaming it.

Reply:
+1

Reply:
Done :+1: